Node/files/node.py

```diff
@@ -98,7 +99,14 @@ def get_localhost_of_host():
 def send_data(node_data):
     try:
         hosts_localhost = get_localhost_of_host()
-        requests.post(hosts_localhost, json=node_data, headers={'content-type':'application/json'}, timeout = 1)
+        #requests.post(hosts_localhost, json=node_data, headers={'content-type':'application/json'}, timeout = 1)
```

Comment:
Remove this unneeded code

Reply:
Already done in e2bda94 - however, I left the get_localhost_of_host()-call as outcommented code, as we might get it to work later on.